<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphFile</class>
 <widget class="QWidget" name="GraphFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Matriz</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>590</width>
             <height>429</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QLabel" name="lblMatrix">
              <property name="font">
               <font>
                <family>Consolas</family>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>[]</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>70</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="title">
          <string>Vértice</string>
         </property>
         <widget class="QPushButton" name="btnAddVertex">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>40</y>
            <width>61</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Adicionar</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btnRemoveVertex">
          <property name="geometry">
           <rect>
            <x>234</x>
            <y>40</y>
            <width>61</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Remover</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="txtVertexID">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>151</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="lblVector_3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>31</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>ID(s)</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>90</height>
          </size>
         </property>
         <property name="title">
          <string>Aresta</string>
         </property>
         <widget class="QPushButton" name="btnRemoveEdge">
          <property name="geometry">
           <rect>
            <x>234</x>
            <y>40</y>
            <width>61</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Remover</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="txtEdges">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>151</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="btnAddEdge">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>40</y>
            <width>61</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Adicionar</string>
          </property>
         </widget>
         <widget class="QLabel" name="lblVector_7">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>81</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Conexão(ões)</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="cbDirection">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>151</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Não dirigido?</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="cbWeight">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>60</y>
            <width>101</width>
            <height>19</height>
           </rect>
          </property>
          <property name="text">
           <string>Informar pesos</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>Outros</string>
         </property>
         <widget class="QGroupBox" name="groupBox_6">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>20</y>
            <width>141</width>
            <height>161</height>
           </rect>
          </property>
          <property name="title">
           <string/>
          </property>
          <widget class="QPushButton" name="btnVector">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>70</y>
             <width>121</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>Exibir vetor</string>
           </property>
          </widget>
          <widget class="QPushButton" name="btnConnected">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>121</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>Verificar se é conexo...</string>
           </property>
          </widget>
          <widget class="QPushButton" name="btnCriticPath">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>40</y>
             <width>121</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>Caminho Crítico</string>
           </property>
          </widget>
         </widget>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>131</width>
            <height>161</height>
           </rect>
          </property>
          <property name="title">
           <string/>
          </property>
          <widget class="QPushButton" name="btnDFS">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>70</y>
             <width>111</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>DFS</string>
           </property>
          </widget>
          <widget class="QLabel" name="lblVector_4">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>21</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>ID</string>
           </property>
          </widget>
          <widget class="QPushButton" name="btnDijkstra">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>40</y>
             <width>111</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>Dijkstra</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="txtSearchID">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>10</y>
             <width>91</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QPushButton" name="btnBFS">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>100</y>
             <width>111</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>BFS</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtVertexID</tabstop>
  <tabstop>btnAddVertex</tabstop>
  <tabstop>btnRemoveVertex</tabstop>
  <tabstop>txtEdges</tabstop>
  <tabstop>cbDirection</tabstop>
  <tabstop>btnAddEdge</tabstop>
  <tabstop>btnRemoveEdge</tabstop>
  <tabstop>scrollArea</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
